src/Components/Color/ColorCard.jsx

```diff
@@ -35,6 +34,19 @@ export default function ColorCard({ color, onDelete, onUpdate }) {
     setIsEditing(false);
   };
 
+  const getContrastMessageStyle = () => {
+    switch (color.contrastStatus) {
+      case "Yup":
+        return { color: "green" };
+      case "Kinda":
+        return { color: "orange" };
+      case "Nope":
+        return { color: "red" };
+      default:
+        return {};
+    }
+  };
```

Comment:
this is a nice usage of a switch statement. There is another way which is a bit more concise. Both are fine though: 
```js
const messageToStyleMap = {
  Yup: "green",
  Kinda: "orange",
  Nope: "red"
}

<p style={{color: messageToStyleMap[color.contrastStatus]}}
```